homeassistant/components/whirlpool/quality_scale.yaml

```diff
@@ -0,0 +1,81 @@
+rules:
```

Comment:
usually we store constants used as keys as `CONF_*` and for some reason you have a map that translates them, but they are also called `CONF_`, not a blocker, but it is interesting

Reply:
Done: https://github.com/home-assistant/core/pull/143409